worker/uniter/uniter.go

```diff
@@ -295,8 +295,8 @@ func (u *Uniter) loop(unitTag names.UnitTag) (err error) {
 			errorString = err.Error()
 		}
 		if errors.Cause(err) == ErrCAASUnitDead {
```

Comment:
Any related change to the ErrCAASUnitDead string?

Reply:
No; unaffected there.